content/doc/book/pipeline/syntax.adoc

```diff
@@ -0,0 +1,851 @@
+hatch." `script` blocks of non-trivial size and/or complexity should be moved
+into <<shared-libraries#, Shared Libraries>> instead.
+
+[[script-example]]
+===== Example
+
+[pipeline]
+----
+// Declarative //
+pipeline {
+    agent any
+    stages {
+        stage('Example') {
+            steps {
+                echo 'Hello World'
+
+                script {
+                    def browsers = ['chrome', 'firefox']
+                    for (int i = 0; i < browsers.size(); ++i) {
+                        echo "Testing the ${browsers[i]} browser"
+                    }
+                }
+            }
+        }
+    }
+}
+----
+
+[role=syntax]
+== Scripted Pipeline
+
+Scripted Pipeline, like <<declarative-pipeline>>, is built on top of the
+underlying Pipeline sub-system. Unlike Declarative, Scripted Pipeline is
+effectively a general purpose DSL
+footnoteref:[dsl,Domain-specific Language]
+built with
+link:http://groovy-lang.org/syntax.html[Groovy].
+Most functionality provided by the Groovy language is made available to users
+of Scripted Pipeline, which means it can be a very expressive and flexible tool
+with which one can author continuous delivery pipelines.
+
+
+=== Flow Control
+
+Scripted Pipeline is serially executed from the top of a `Jenkinsfile`
+downwards, like most traditional scripts in Groovy or other languages.
+Providing flow control therefore rests on Groovy expressions, such as the
+`if/else if/else` conditionals, for example:
+
+[pipeline]
+----
+// Scripted //
+node {
+    stage('Example') {
+        if (env.BRANCH_NAME == 'master') {
+            echo 'I only execute on the master branch'
+        }
+    }
+}
+----
+
+Another way Scripted Pipeline flow control can be managed is with Groovy's
+exception handling support. When <<scripted-steps>> fail for whatever reason
+they throw an exception.  Meaning handling on-error behaviors must make use of
+the `try/catch/finally` blocks in Groovy, for example:
+
+[pipeline]
+----
+// Scripted //
+node {
+    stage('Example') {
+        try {
+            sh 'exit 1'
+        }
+        catch (exc) {
+            echo 'Something failed, I should sound the klaxons!'
+            throw
+        }
+    }
+}
+----
+
+
+[[scripted-steps]]
+=== Steps
+
+As discussed in <<getting-started#, Getting Started>>, the most fundamental
+part of a Pipeline is the "step." Fundamentally, steps tell Jenkins _what_ to
+do, and serve as the basic building block for both Declarative and Scripted
+Pipeline syntax.
+
+Scripted Pipeline does *not* introduce any steps which are specific to its
+syntax; 
+link:/doc/pipeline/steps[Pipeline Steps reference]
+which contains a comprehensive list of steps built into Pipeline as well as
+steps provided by plugins. 
+
+
+=== Differences from plain Groovy
+
+////
+XXX: TODO https://issues.jenkins-ci.org/browse/WEBSITE-267
+https://issues.jenkins-ci.org/browse/WEBSITE-289
+////
+
+In order to provide durability, meaning running Pipelines can survive a restart
```

Comment:
I'd emphasise "durability" and re-word slightly: 

"In order to provide \<i>durability\</i>, which means that running Pipelines can survive…"